backend/src/main/java/ca/gov/dtsstn/cdcp/api/service/UserService.java

```diff
@@ -92,14 +98,17 @@ public ConfirmationCode createConfirmationCodeForUser(String userId) {
 	public Subscription createSubscriptionForUser(String userId, Subscription subscription) {
 		Assert.hasText(userId, "userId is required; it must not be null or blank");
 		Assert.hasText(subscription.getAlertType().getCode(), "subscription.alertType.code is required; it must not be null or blank");
-		Assert.notNull(subscription.getPreferredLanguage(), "subscription.preferredLanguage is required; it must not be null");
+		Assert.notNull(subscription.getPreferredLanguage().getCode(), "subscription.preferredLanguage.code is required; it must not be null");
```

Comment:
It's possible that the caller of this function can pass in a null language, in which case the call to `.getPreferredLanguage().getCode()` will result in a null pointer exception.

(I also just realized this is true for the alertType check above)